.github/workflows/test_qdrant.yml

```
@@ -14,6 +16,8 @@
   RUNTIME__LOG_LEVEL: ERROR
 
 jobs:
+  if: ${{ github.event.label.name == 'run-checks' }}
```

actionlint

"steps" section is missing in job "if"

(syntax-check)